src/taipy/core/data/excel.py

```diff
@@ -48,8 +49,8 @@ class ExcelDataNode(DataNode):
             `None`.
         parent_ids (Optional[Set[str]]): The identifiers of the parent tasks or `None`.
         last_edit_date (datetime): The date and time of the last modification.
-        job_ids (List[str]): The ordered list of jobs that have written this data node.
-        version (str): The string indicates the application version of the data node to instantiate. If not provided, the latest version is used.
+        edits (List[Edit]): The ordered list of edits for that job.
```

Comment:
This is post-process by some documentation tool. the `^` character is a hook to look so the post-processing knows it must add a link to the documentation page of the `Entry` object

```suggestion
        edits (List[Edit^]): The ordered list of edits for that job.
```